packages/usa-footer/src/styles/_usa-footer.scss

```diff
@@ -22,103 +22,102 @@ $-chevron-expand-more: map-merge(
+}
 
-    // Gaps
-    &.grid-gap {
-      @include grid-gap;
-    }
+.grid-gap-2 {
+  @include grid-gap(2);
+}
 
-    &.grid-gap-1 {
-      @include grid-gap(1);
-    }
+.grid-gap-4 {
+  @include grid-gap(4);
+}
 
-    &.grid-gap-2 {
-      @include grid-gap(2);
-    }
+// Gaps: Mobile Large
+/* stylelint-disable */
+@include at-media("mobile-lg") {
+  .mobile-lg\:grid-gap-2 {
+    @include grid-gap(2);
+  }
+}
+/* stylelint-enable */
 
-    &.grid-gap-4 {
-      @include grid-gap(4);
-    }
+// Columns
+[class*="grid-col"] {
+  @include u-position(relative);
+  @include u-width(full);
+  box-sizing: border-box;
+}
 
-    // Gaps: Mobile Large
-    /* stylelint-disable */
-    @include at-media("mobile-lg") {
-      &.mobile-lg\:grid-gap-2 {
-        @include grid-gap(2);
-      }
-    }
-    /* stylelint-enable */
+.grid-col-auto {
+  @include grid-col("auto");
+}
 
-    // Columns
-    [class*="grid-col"] {
-      @include u-position(relative);
-      @include u-width(full);
-      box-sizing: border-box;
-    }
+// Columns: Mobile Large
+/* stylelint-disable */
+@include at-media("mobile-lg") {
+  .mobile-lg\:grid-col-auto {
+    @include grid-col("auto");
+  }
 
-    .grid-col-auto {
-      @include grid-col("auto");
-    }
+  .mobile-lg\:grid-col-4 {
+    @include grid-col(4);
+  }
 
-    // Columns: Mobile Large
-    /* stylelint-disable */
-    @include at-media("mobile-lg") {
-      .mobile-lg\:grid-col-auto {
-        @include grid-col("auto");
-      }
+  .mobile-lg\:grid-col-6 {
+    @include grid-col(6);
+  }
 
-      .mobile-lg\:grid-col-4 {
-        @include grid-col(4);
-      }
+  .mobile-lg\:grid-col-8 {
+    @include grid-col(8);
+  }
 
-      .mobile-lg\:grid-col-6 {
-        @include grid-col(6);
-      }
+  .mobile-lg\:grid-col-12 {
+    @include grid-col(12);
+  }
+}
 
-      .mobile-lg\:grid-col-8 {
-        @include grid-col(8);
-      }
+// Columns: Tablet
+@include at-media("tablet") {
+  .tablet\:grid-col-4 {
+    @include grid-col(4);
+  }
 
-      .mobile-lg\:grid-col-12 {
-        @include grid-col(12);
-      }
-    }
+  .tablet\:grid-col-8 {
+    @include grid-col(8);
+  }
+}
 
-    // Columns: Tablet
-    @include at-media("tablet") {
-      .tablet\:grid-col-4 {
-        @include grid-col(4);
-      }
+// Columns: Desktop
+@include at-media("desktop") {
+  .desktop\:grid-col-auto {
+    @include grid-col("auto");
+  }
 
-      .tablet\:grid-col-8 {
-        @include grid-col(8);
-      }
-    }
+  .desktop\:grid-col-3 {
+    @include grid-col(3);
+  }
+}
 
-    // Columns: Desktop
-    @include at-media("desktop") {
-      .desktop\:grid-col-auto {
-        @include grid-col("auto");
-      }
+// General footer styles
+.usa-footer {
+  @include border-box-sizing;
+  @include typeset($theme-footer-font-family);
+  overflow: hidden;
 
-      .desktop\:grid-col-3 {
-        @include grid-col(3);
-      }
-    }
-    /* stylelint-enable */
+  > .grid-container {
+    @include grid-container($theme-footer-max-width);
```

Comment:
Moving `.grid-container` class to match what it was in `3.7.0` before changes were made.

https://github.com/uswds/uswds/blob/e670ce36a7ce6179fd789cf2ae1d1feae955f83a/packages/usa-footer/src/styles/_usa-footer.scss#L26-L34

Reply:
@mahoneycm to confirm, the current state has them outside and this change brings them back under footer class?

Reply:
The current state has _all_ of the grid classes nested under the footer class. This change moves them outside of the footer class to resolve specificity issues.

I maintained `.usa-footer > .grid-container` to match `3.7.0`